tdrs-backend/tdpservice/email/email.py

```diff
@@ -0,0 +1,88 @@
+"""Send emails."""
+
+from celery import shared_task
+from django.core.exceptions import ValidationError
+from django.core.validators import validate_email
+from django.core.mail import EmailMultiAlternatives
+from django.conf import settings
+from django.template.loader import get_template
+from django.template import TemplateDoesNotExist
+from tdpservice.email.email_enums import EmailType
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+@shared_task
+def construct_email(email_type, context):
+    """Get email template."""
+    template_path = email_type.value + ".html"
```

Comment:
adding '.html' to the end of the enum value, but maybe that should be included in the enum.

Reply:
personally, i'd prefer the `.html` with the template name in the enum